exporter/src/main/java/io/zeebe/exporter/record/value/MessageRecordValue.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.exporter.record.value;
+
+import io.zeebe.exporter.record.RecordValueWithPayload;
+
+/**
+ * Represents a message correlation related event or command.
+ *
+ * <p>See {@link io.zeebe.protocol.intent.MessageIntent} for intents.
+ */
+public interface MessageRecordValue extends RecordValueWithPayload {
```

Comment:
New field is added in the meantime: "getTimeToLive()" Returns the lifetime of the message.